src/test/java/org/obiba/es/mica/query/RQLQueryTest.java

```diff
@@ -232,7 +232,7 @@ public void test_rql_query_range() throws IOException {
   @Test
   public void test_rql_query_match() throws IOException {
```

Comment:
Why replacing the test? why not adding new ones to test the new RQL function?
Make sure the test function name explains what the test is about.